pnpm-lock.yaml

```diff
@@ -93,9 +93,9 @@ importers:
       vanilla-cookieconsent: 2.8.5
     devDependencies:
       '@storybook/addon-a11y': 6.5.12_biqbaboplfbrettd7655fr4n2y
-      '@storybook/addon-essentials': 6.5.12_ehxjltspajcysklsgue4g44hsy
-      '@storybook/builder-vite': 0.2.2_7dwu2gf4q36hnlqkbgbkgqupdm
-      '@storybook/react': 6.5.12_ehxjltspajcysklsgue4g44hsy
+      '@storybook/addon-essentials': 6.5.12_txkhxnwx7iliil3hgqqpsbtdkq
```

Comment:
@Jordan-Mysten these are generated changes when I ran below `sui/`, is it expected?
```
rm -rf ./node_modules && rm -rf ./wallet/node_modules/
pnpm install
```